docs/src/security/tutorial007.py

```diff
@@ -8,7 +10,9 @@
 
 
 def get_current_username(credentials: HTTPBasicCredentials = Depends(security)):
-    if credentials.username != "foo" or credentials.password != "password":
+    # Be careful to use constant time comparison to prevent timing attacks
+    if (!hmac.compare_digest(credentials.username, "foo") or
```

Comment:
shouldn't this be a `not hmac.compare_digest` rather than using `!`?

Reply:
Yes, good point. I haven't been doing much Python lately.